### PR TITLE
Introduced vjoinsplit in the decoderawtransaction's output

### PR DESCRIPTION
**Pull request by MarcoOl94**
Introduced vjoinsplit in the decoderawtransaction's output

### CODE REVIEW

File: src/rpc/rawtransaction.cpp [hunk 1]
```diff
@@ -237,6 +237,9 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
     }
     entry.push_back(Pair("vout", vout));
 
+    UniValue vjoinsplit = TxJoinSplitToJSON(tx);
+    entry.push_back(Pair("vjoinsplit", vjoinsplit));
+
```
Comment (abi87) [inline src/rpc/rawtransaction.cpp]: I understand the change and it looks fine to me. However I do not see any extra unit test/qa tests covering the change you made. I saw cronic opened issue 173 to ask for extra tests (it is marked as enhancement).
@MarcoOl94, @alsala  should we add the unit tests or qa tests covering this change?

Comment (abi87) [inline src/rpc/rawtransaction.cpp:242]: I understand the change and it looks fine to me. However I do not see any extra unit test/qa tests covering the change you made. I saw cronic opened issue 173 to ask for extra tests (it is marked as enhancement).
@MarcoOl94, @alsala should we add the unit tests or qa tests covering this change?

**Review by abi87 — CHANGES_REQUESTED** [review]
Once we agree on whether unit/qa tests should go with the change, I can approve the change

File: src/rpc/rawtransaction.cpp (hunk 1 above)
Comment (cronicc) [reply src/rpc/rawtransaction.cpp:242]: Issue #173 is about adding unit tests to verify that the output of all RPC calls has the expected format and includes all values that should be there, not just decoderawtransaction.
That clearly is a larger task and deserves it's own PR.

For this PR no extra tests are needed IMO.
LGTM

Comment (abi87) [reply src/rpc/rawtransaction.cpp:242]: Thanks for clarifying how the task splitting was done @cronicc .
Approving the change

**Review by abi87 — APPROVED** [review]

**Review by alsala — APPROVED** [review]

**Review by albertog78 — APPROVED** [review]

**Pull request merged**